website/docs/configuration.md

```
@@ -82,16 +68,16 @@ Lorsque `tmdb_token` est configuré, les résolveurs **TMDB et IMDB** sont autom
+### Relais Nostr
 
 | Paramètre | Type | Défaut | Description |
 |-----------|------|--------|-------------|
-| `ygg_domain` | string | `null` | Domaine YGG personnalisé (optionnel) |
+| `relay_url` | string | `wss://relay.ygg.gratis` | URL du relais Nostr |
 
-:::tip Quand utiliser YGG_DOMAIN ?
-Par défaut, Ygégé détecte automatiquement le domaine YGG actuel via une redirection depuis `ygg.re`. Si cette auto-détection échoue (erreurs 307, "Session expired..."), vous pouvez spécifier manuellement le domaine :
+:::tip Quand utiliser RELAY_URL ?
+Par défaut, Ygégé se connecte au relais officiel `wss://relay.ygg.gratis`. Si vous souhaitez utiliser un relais alternatif ou un miroir, spécifiez son URL WebSocket :
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)